bin/spack.bat

```diff
@@ -154,20 +171,25 @@ goto :end_switch
 if NOT defined _sp_args (
     goto :default_case
 )
-set args_no_quote=%_sp_args:"=%
-if NOT "%args_no_quote%"=="%args_no_quote:--help=%" (
+
+if NOT "%_sp_args%"=="%_sp_args:--help=%" (
+    echo "--help"
```

Comment:
(question) what is the purpose of `echo`ing this?

Reply:
Oh I think that may be debug cruft I left in by accident. Same for the other `echo`s below.